scripts/unixbench/dependencies.txt

```diff
@@ -0,0 +1,26 @@
+### START - Dependency installation order ###
+unixbench_order = 50
+### END - Dependency installation order ###
+
+### START - Dependency-specific installation method ###
+# pm = "package manager" (yum or apt-get)
+# sl = "soft link" (assume that the dependency is already installed, just has to
+# be properly exposed to the user's path.
+# git = git clone using above giturl
+# pip = python pip utility
+# man = "manual"
+linpack-install = man
+### END - Dependency-specific installation method ###
+
+### START - Tests ###
+### END - Tests ###
+
+### START - Dependency versions ###
+unixbench-ver = ANY
+### END - Dependency versions ###
+
+### START - Dependency URLs ###
```

Comment:
This is not complete. It should include a specific set of accurately-crafted commands that any user can copy-and-paste to make their local environment benchmark-ready.

Even if you're using "man" option, the URLs still need to be there for user references purposes.